src/styles/vars.scss

```diff
@@ -0,0 +1,20 @@
+// Sizes
+$navbar-width: 250px;
+$page-width: calc(100% - $navbar-width);
+$page-padding: 50px;
+
+// Colors
+// Got these color names from https://www.color-name.com/hex.
+$color-ufo-green: #22c55e;
+$color-orange: #f59e0b;
+$color-carmine-pink: #ef4444;
+$color-solid-black: #161616;
+$color-raisin-black: #262626;
+$color-chinese-white: #e1e1e1;
+$color-gunmetal: #28343f;
+
+// Colors - Background & Text
+$bg-color-game-card: $color-raisin-black;
```

Comment:
I mentioned this in a DM, but I'd like to see these variable names be more generic. I'm not looking at the colors to know which one is used more than other, but in general assign the more used one to the base name, and lesser used ones to `alt`.

This is completely for example, do whatever makes the most sense in the field:
```
// Colors - Background & Text
$bg-dark: $color-raisin-black;
$text-dark: $color-solid-black;
$text-dark-alt: $color-gunmetal; // moved this one up a line to organize it beside the other dark variable for text
$text-light: $color-chinese-white;

```